context/AuthProvider.tsx

```diff
@@ -0,0 +1,110 @@
+import React, { createContext, useState } from "react";
+import axiosConfig, { setUserToken } from "../helpers/axios/axiosConfig";
```

Comment:
We normally just call the general  axios object `axios`
```suggestion
import axios, { setUserToken } from "../helpers/axios/axiosConfig";
```

Reply:
Thanks for the suggestion, done!